build/build.sh

```diff
@@ -0,0 +1,15 @@
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+export https_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+if [ "$use_remote" = "1" ]; 
+    then 
+        cd /tmp/
+        git clone https://github.com/secure-device-onboard/iot-platform-sdk 
```

Comment:
Create a top-level variable for repo and branch name so that developers can easily change this to use their forked copies.

REMOTE_URL=https://github.com/secure-device-onboard/iot-platform-sdk
REMOTE_BRANCH=master

git clone $REMOTE_URL
cd /tmp/iot-platform-sdk
git checkout $REMOTE_BRANCH

Reply:
Done.